src/Dapr.Client/DaprClient.cs

```diff
@@ -978,16 +978,16 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
         /// <summary>
         /// Attempt to start the given workflow with response indicating success.
         /// </summary>
-        /// <param name="instanceId">Identifier of the specific run.</param>
+        /// <param name="instanceID">Identifier of the specific run.</param>
         /// <param name="workflowComponent">The component to interface with.</param>
         /// <param name="workflowName">Name of the workflow to run.</param>
         /// <param name="workflowOptions">The list of options that are potentially needed to start a workflow.</param>
         /// <param name="input">The input input for the given workflow.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
-        /// <returns>A <see cref="Task"/> containing a <see cref="WorkflowReference"/></returns>
+        /// <returns>A <see cref="Task"/> containing a <see cref="StartWorkflowResponse"/></returns>
         [Obsolete("This API is currently not stable as it is in the Alpha stage. This attribute will be removed once it is stable.")]
-        public abstract Task<WorkflowReference> StartWorkflowAsync(
-            string instanceId,
+        public abstract Task<StartWorkflowResponse> StartWorkflowAsync(
+            string instanceID,
```

Comment:
Nit: all the other workflow methods use the original `instanceId`.

Reply:
@RyanLettieri let's not do the Id --> ID rename here. In C#, it's common practice to use "Id" instead of "ID" (at one point, I think some linters even required this).